src/main/resources/migrations/00048_create_table_stripe_agreements.sql

```diff
@@ -0,0 +1,10 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:create_table-stripe_agreements
+CREATE TABLE stripe_agreements (
+  id SERIAL PRIMARY KEY,
+  service_id INT NOT NULL REFERENCES services (id),
```

Comment:
is it worth having a unique index on service_id (if it should be unique that is)? is this something we do elsewhere?